src/Microsoft.AspNet.Mvc.ViewFeatures/Internal/ValidationHelpers.cs

```diff
@@ -3,27 +3,45 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using Microsoft.AspNet.Mvc.ModelBinding;
 
 namespace Microsoft.AspNet.Mvc.ViewFeatures.Internal
 {
-    internal static class ValidationHelpers
+    public static class ValidationHelpers
```

Comment:
You want the class to be renamed as well? Or are you objecting to something else?